src/js/text/main.html

```diff
@@ -82,11 +82,8 @@
             We release a new documentary every month.<br>
             Would you like a reminder with new releases?
           </p>
-          <form class='email-form__form'>
-            <label for='email' class='email-address'>Email address</label>
-            <input type='email' id='email' placeholder="Email address">
-            <input type='submit' id='remind-me' value='Remind me'>
-          </form>
+
+            <iframe height="131px" data-form-success-headline="Thank you" data-form-success-desc="We will send you reminders about forthcoming documentaries." scrolling="no" seamless="" frameborder="0" class="iframed--overflow-hidden email-sub__iframe email-sub__iframe--end-article js-email-sub__iframe js-email-sub__iframe--article"></iframe>
```

Comment:
Looks like this got depreciated so will remove: http://caniuse.com/#feat=iframe-seamless